integration/tests/navigation.test.ts

```diff
@@ -91,129 +91,130 @@ export default function Page() {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.waitForMounted();
+    await u.po.signIn.signInWithEmailAndInstantPassword({ email: fakeUser.email, password: fakeUser.password });
+    await u.po.expect.toBeSignedIn();
+
+    await u.page.goToRelative('/hash/user');
+    await u.po.userProfile.waitForMounted();
+
+    await u.po.userProfile.clickSetUsername();
+
+    u.page.getByText(/Update username/i);
+
+    await u.po.userProfile.typeUsername('some_username');
+
+    await u.page.getByText(/Cancel/i).click();
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+
+    u.page.getByText(/an email containing/i);
+
+    await u.po.userProfile.typeEmailAddress('some@email.com');
+
+    await u.page.getByText(/Cancel/i).click();
+  });
+
+  test('sign in with path routing', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.waitForMounted();
+
+    await u.po.signIn.setIdentifier(fakeUser.email);
+    await u.po.signIn.continue();
+    await u.page.waitForURL(`${app.serverUrl}/sign-in/factor-one`);
+
+    await u.po.signIn.setPassword(fakeUser.password);
+    await u.po.signIn.continue();
+
+    await u.po.expect.toBeSignedIn();
+  });
+
+  test('sign in with hash routing', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.page.goToRelative('/hash/sign-in');
+    await u.po.signIn.waitForMounted();
+
+    await u.po.signIn.setIdentifier(fakeUser.email);
+    await u.po.signIn.continue();
+    await u.page.waitForURL(`${app.serverUrl}/hash/sign-in#/factor-one`);
+
+    await u.po.signIn.setPassword(fakeUser.password);
+    await u.po.signIn.continue();
+
+    await u.po.expect.toBeSignedIn();
+  });
+
+  test('user profile from user button navigates correctly', async ({ page, context }) => {
+    const u = createTestUtils({ app, page, context });
+    await u.po.signIn.goTo();
+    await u.po.signIn.waitForMounted();
+    await u.po.signIn.signInWithEmailAndInstantPassword({ email: fakeUser.email, password: fakeUser.password });
+    await u.po.expect.toBeSignedIn();
+
+    await u.page.goToRelative('/');
+    await u.page.waitForClerkComponentMounted();
+
+    await u.page.getByRole('button', { name: 'Open user button' }).click();
+
+    await u.page.getByText(/Manage account/).click();
+
+    await u.page.waitForSelector('.cl-modalContent > .cl-userProfile-root', { state: 'attached' });
+
+    await u.po.userProfile.clickSetUsername();
+    await u.page.getByText(/Cancel/i).click();
+
+    await u.page.waitForSelector('.cl-profileSectionContent__username .cl-headerTitle', { state: 'detached' });
+
+    await u.po.userProfile.clickAddEmailAddress();
+    await u.page.getByText(/Cancel/i).click();
```

Comment:
This test no longer tests navigation but interaction within the component. Should this be moved ?